src/KeyVault/KeyVault/Track2Models/Track2VaultClient.cs

```diff
@@ -221,17 +221,26 @@ internal PSKeyVaultCertificate ImportCertificate(string vaultName, string certNa
                 throw new ArgumentNullException(nameof(certificate));
 
             var certClient = CreateCertificateClient(vaultName);
-            return ImportCertificate(certClient, certName, certificate, password, tags, contentType);
+            return ImportCertificate(certClient, certName, certificate, password, tags, contentType, certPolicy);
         }
 
-        private PSKeyVaultCertificate ImportCertificate(CertificateClient certClient, string certName, byte[] certificate, SecureString password, IDictionary<string, string> tags, string contentType = Constants.Pkcs12ContentType)
+        private PSKeyVaultCertificate ImportCertificate(CertificateClient certClient, string certName, byte[] certificate, SecureString password, IDictionary<string, string> tags, string contentType = Constants.Pkcs12ContentType, PSKeyVaultCertificatePolicy certPolicy = null)
         {
-            var options = new ImportCertificateOptions(certName, certificate)
+            CertificatePolicy certificatePolicy = null;
+            if (certPolicy == null)
             {
-                Policy = new CertificatePolicy()
+                certificatePolicy = new CertificatePolicy()
                 {
                     ContentType = contentType
-                },
+                };
+            }
+            else
+            {
+                certificatePolicy = certPolicy.ToTrack2CertificatePolicy();
```

Comment:
ContentType is missing if certPolicy exists, please have a look

Reply:
Discussion Note: user provided `-ContentType` should overwrite the ContentType in certPolicy.